proto/channel.proto

```diff
@@ -12,7 +12,7 @@ message Channel {
     repeated Entity cc = 5; // gives the entries of the CC (carbon copy) field of the email.
     repeated Entity bcc = 6; // gives the entries of the BCC (blind carbon copy) field of the email.
     float min_period = 7; // gives the minimum push period frequency for a channel, in seconds.
-    int64 max_size = 8; // gives the maximum allowed size of the request, in bytes.
+    int max_size = 8; // gives the maximum allowed size of the request, in bytes.
```

Comment:
changed because otherwise the int64 would be treated as a string. Since in JSON int64 is not supported, I thought we might as well remove it (here and in the swagger). I thought it's not likely that the `max_size` is ever larger than a few megabites, let alone max_signed_int (> 2 gigabytes).

Reply:
Sounds good to me.